src/server/pegasus_write_service_impl.h

```diff
@@ -806,6 +833,23 @@ class pegasus_write_service::impl : public dsn::replication::replica_base
         return expire_ts;
     }
 
+    dsn::error_code get_external_files_path(const std::string &bulk_load_dir,
```

Comment:
Should be a global function.

Reply:
Why should this function be a global function?

Reply:
Because it doesn't touch any member of this class. It's merely a util. Setting it global helps shortening the size of pegasus_write_service_impl.

Reply:
okay